src/esp/bindings/GfxBindings.cpp

```diff
@@ -57,15 +57,15 @@ void initGfxBindings(py::module& m) {
   py::class_<Renderer, Renderer::ptr>(m, "Renderer")
       .def(py::init(&Renderer::create<>))
       .def("draw",
-           py::overload_cast<sensor::VisualSensor&, scene::SceneGraph&, bool>(
-               &Renderer::draw),
+           py::overload_cast<sensor::VisualSensor&, scene::SceneGraph&, bool,
+                             bool>(&Renderer::draw),
            R"(Draw given scene using the visual sensor)", "visualSensor"_a,
-           "scene"_a, "frustumCulling"_a = true)
+           "scene"_a, "frustumCulling"_a = true, "objectsOnly"_a = false)
```

Comment:
As we talked about yesterday, instead of passing individual booleans (which may increase in number and impact a large amount of code) I would suggest we approach these draw state-related boolean arguments as a single int.  We define an enum of possible flag settings, pybind the enum and perhaps a 2 line helper function for taking a list of enum values and or'ing them together to produce the state int encoding, and then use the enum values anded with the state int encoding when we wish to consume them.  

This way, we can grow to use up to 32 booleans without requiring any refactoring of method headers or bindings.  As a user, too, i would not want to always have to enter all the flags on method call if i did not need them.

Reply:
On @jturner65's note here, we should definitely use kwargs for call the python side if we don't do the ENUM thing.
```suggestion
           "scene"_a, "frustum_culling"_a = true, "objects_only"_a = false)
```

Then we can do
```python
self._sim.renderer.draw(
                 self._sensor_object,
                 self._sim.get_active_scene_graph(),
                 frustum_culling=self._sim.frustum_culling,
                 objects_only=True,
             )
```

I am also a fan of doing kwargs in cpp via comments, i.e.
```cpp
/*frustumCulling=*/sim.isFrustumCullingEnabled(), /*objectsOnly=*/true
```

Reply:
You can also use [Containers::EnumSet](https://doc.magnum.graphics/corrade/classCorrade_1_1Containers_1_1EnumSet.html) and bind it to a `py::enum_`, which will make it more typesafe. Check Magnum Python bindings sources for how this could be done (e.g. the shader flags), there are also a bunch of utils for adding bitwise operators to enum types.

Reply:
> kwargs

Does anybody know if it's possible to *force* pybind arguments to be keyword-only? Something equivalent to the `*` in `draw(object, scene, *, frustum_culling, objects_only)`. I know only that if one doesn't annotate the arguments with `_a`, those are treated as position-only (equivalent to a `draw(object, scene, /)` signature).

Reply:
I took a pass adding EnumSet following the example of the phong flags. Builds fine and C++ tests pass, but getting a python module issue. Anything glaringly wrong here @mosra?

Reply:
> force pybind arguments to be keyword-only

I don't think so -- I did a little looking as I _love_ keyword-only args and didn't find anything